roles/caw/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+
+- name: Fetch CAW from GitHub
+  git: repo="{{ caw_repo }}"
+       dest="{{ caw_dest }}"
+       version="{{ caw_version }}"
+       force=yes
+
+- name: Create CAW Singularity folder
+  file: name="{{ caw_singularity }}"
+        state=directory mode=g+s
+
+- name: Create temp directory for CAW singularity images
+  command: "mkdir {{ item }}"
```

Comment:
Any specific reason for why you're using `mkdir` as a shell command instead of using `file` with `state=directory`?

Reply:
Initially I used `/scratch` for `singularity_cache` and `singularity_temp` and also I did not cleanup. Thought `file` module dint have `creates` arg so used `command`.

But later realised it (the cache and temp directories) requires a lot of space and changed their path within new deployment env itself (as I would cleanup them later). Then it did not strike me to use `file` but yes I can change it 👍